spec/index.html

```diff
@@ -248,5 +248,36 @@ <h2>Basic Concepts</h2>
 
     <p>FIXME.</p>
   </section>
- </body>
+
+  <section id="sec" class="informative">
+    <h2>Security Considerations</h2>
+
+    <p>See the YAML media type registration.</p>
+  </section>
```

Comment:
IANA Security considerations should reference this section.

Reply:
> IANA Security considerations should reference this section.

That makes it inconsistent with how it's done in JSON-LD. That may be fine if we're internally consistent, but it is different.

Reply:
In the past I have been suggested to avoid stuffing things into IANA considerations, because the space of the specification is the specification document. For example, see https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-binary-message-05#section-7 .

In particular, I have been pointed to https://datatracker.ietf.org/doc/html/rfc8126#section-1.1

> The purpose of having a dedicated IANA Considerations section is to
   provide a single place to collect clear and concise information and
   instructions for IANA.  Technical documentation should reside in
   other parts of the document; the IANA Considerations should refer to
   these other sections by reference only (as needed).

Reply:
Okay, wish we had known that when we did the JSON-LD IANA considerations, of course, that was way back in the 1.0 timeframe in 2014, so we should go with your division. We'll need to follow up after both PRs are merged. (I usually wait a week for comments to come in before merging with the consensus opinions).

Reply:
Honestly, I didn't know that either:) that's all stuff I learned thanks to the reviews of the httpwg folks :)))